tests/basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/cache/container/distributed/cache/DistributedCacheFinderTest.java

```diff
@@ -0,0 +1,94 @@
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+import static org.jboss.hal.dmr.ModelDescriptionConstants.INFINISPAN;
+import static org.jboss.hal.testsuite.fragment.finder.FinderFragment.configurationSubsystemPath;
+import static org.jboss.hal.testsuite.test.configuration.infinispan.InfinispanFixtures.cacheContainerAddress;
+import static org.jboss.hal.testsuite.test.configuration.infinispan.InfinispanFixtures.distributedCacheAddress;
+
+@RunWith(Arquillian.class)
+public class DistributedCacheFinderTest {
+
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final Operations operations = new Operations(client);
+    private static final Administration administration = new Administration(client);
+
+    private static final String CACHE_CONTAINER = "cache-container-" + Random.name();
+    private static final String DISTRIBUTED_CACHE_CREATE = "distributed-cache-create-" + Random.name();
+
+    @BeforeClass
+    public static void setUp() throws IOException, TimeoutException, InterruptedException {
+        operations.add(cacheContainerAddress(CACHE_CONTAINER));
+        administration.reloadIfRequired();
+    }
+
+    @AfterClass
+    public static void tearDown() throws IOException, OperationException {
+        try {
+            operations.removeIfExists(cacheContainerAddress(CACHE_CONTAINER));
+        } finally {
+            client.close();
+        }
+    }
+
+    @Drone
+    private WebDriver browser;
+
+    @Inject
+    private Console console;
+
+    private ColumnFragment cacheColumn;
+
+    @Before
+    public void initPage() {
+        cacheColumn = console.finder(NameTokens.CONFIGURATION,
+                configurationSubsystemPath(INFINISPAN).append(Ids.CACHE_CONTAINER, Ids.cacheContainer(CACHE_CONTAINER)))
+                .column("cache");
+    }
+
+    @Test
+    public void create() throws Exception {
+        cacheColumn.dropdownAction("cache-add-actions", "distributed-cache-add");
+        AddResourceDialogFragment addResourceDialogFragment = console.addResourceDialog();
+        addResourceDialogFragment.getForm().text("name", DISTRIBUTED_CACHE_CREATE);
+        addResourceDialogFragment.add();
+        try {
+            console.verifySuccess();
+        } catch (org.openqa.selenium.TimeoutException e) {
+            Assert.fail("Newly distributed cache isn't created successfully. Probably due to https://issues.jboss.org/browse/HAL-1509.");
```

Comment:
I suggest to remove the try-catch block here. If the `TimeoutException` occurs, there will be test failure anyway and the code from line n.85 won't still be executed.